scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
Presumably this script will need to be rolled into the repo manager, right? We want to say, add-feature-set <gff file> and the repo manager parses the GFF file and creates the SQLlite DB. The repository administrator shouldn't need to know anything about our internal DB format for storing features.